gradle.properties

```diff
@@ -13,8 +13,8 @@ POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=seatgeek
 POM_DEVELOPER_NAME=SeatGeek, Inc
 
-ANDROID_PLUGIN_VERSION=1.3.0
-BUILD_TOOLS_VERSION=21.1.2
-SUPPORT_LIBRARY_VERSION=22.2.1
-TARGET_SDK_VERSION=21
+ANDROID_PLUGIN_VERSION=2.3.3
+BUILD_TOOLS_VERSION=26.0.0
+SUPPORT_LIBRARY_VERSION=25.4.0
```

Comment:
I read about the issues that you're describing and they shouldn't impact this project. I think we're safe to upgrade.